cmd/thanos/query_frontend.go

```diff
@@ -82,6 +83,10 @@ func registerQueryFrontend(app *extkingpin.App) {
 	cmd.Flag("query-frontend.log-queries-longer-than", "Log queries that are slower than the specified duration. "+
 		"Set to 0 to disable. Set to < 0 to enable on all queries.").Default("0").DurationVar(&cfg.CortexFrontendConfig.LogQueriesLongerThan)
 
+	cmd.Flag("query-frontend.org-id-header", "Request header names used to set the org id field in the slow query log (repeated flag). "+
```

Comment:
I am thinking about having a default org-id-header `X-Scope-OrgId` and users can also specify other headers as well. WDYT?

Reply:
Yeah, sounds like a sensible default :+1:.  Will that help with compatibility with Cortex things perhaps?